src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -67,13 +78,42 @@ export const InlineEditExample = {
       demo: <InlineEditTitle />,
       props: { EuiInlineEditTitle },
     },
+    {
+      title: 'Customizing read and edit modes',
+      text: (
+        <>
+          <p>
+            Customize the <EuiCode>readMode</EuiCode> empty button by passing{' '}
+            <strong>EuiInlineEdit</strong> the <EuiCode>readModeProps</EuiCode>{' '}
+            property. <EuiCode>readMode</EuiCode> accepts{' '}
+            <EuiCode>EuiButtonEmpty</EuiCode> properties with the exception of{' '}
```

Comment:
We should make this a link instead of a code: 
```suggestion
            <Link to="/navigation/button#empty-button"><strong>EuiButtonEmpty</strong></Link> properties with the exception of{' '}
```

Link import comes from react-router-dom:

```
import { Link } from 'react-router-dom';
```